src/components/layout/Sidebar/SidebarItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+import IconButton from '../../core/IconButton';
+import { Link } from 'react-router-dom';
+import Icon from '../../core/Icon';
+
+interface SidebarItemProps {
+  icon: FC<React.SVGProps<SVGSVGElement>>;
+  to: string;
+  tooltip: string;
+  disabled?: boolean;
+}
+
+const SidebarItem: FC<SidebarItemProps> = ({ icon, to, tooltip, disabled }) => {
+  return (
+    <div id="sidebar-item">
```

Comment:
Setting static ids on items whish will be repeated it is not good idea. Some browsers can complain about this.